storage/bucket.go

```diff
@@ -376,23 +376,29 @@ const (
 	// not set in a call to GCS.
 	PublicAccessPreventionUnknown PublicAccessPrevention = iota
 
-	// PublicAccessPreventionUnspecified corresponds to a value of "unspecified"
-	// and is the default for buckets.
+	// PublicAccessPreventionUnspecified corresponds to a value of "inherited".
```

Comment:
this should still be unspecified i think

Reply:
+1, keep `unspecified` here.